ghost/core/core/server/services/email-analytics/jobs/fetch-latest/index.js

```diff
@@ -47,4 +47,7 @@ if (parentPort) {
             process.exit(0);
         }, 1000);
     }
-})();
+})().catch((e) => {
+    console.error(e); // eslint-disable-line no-console
```

Comment:
The errors thrown in jobs are handled [by this code](https://github.com/TryGhost/Ghost/blob/520b19e31376778a068165c34dac12cd8d6ae988/ghost/core/core/server/services/jobs/job-service.js#L13-L15). 

Error handling is part of the bree instance options - https://github.com/breejs/bree/tree/4f388fe70c1c27293ecff0bd6c8f8bd8a649405b#instance-options (afair we added this feature 😊 )

Reply:
I'll remove these

Reply:
Done! ✅